### ISSUE TITLE
Incorrect object on viewport help

### ISSUE THREAD

**Issue by goncy** [cypress-io/cypress#1350]
![screen shot 2018-02-20 at 10 15 38](https://user-images.githubusercontent.com/6494462/36425956-5547c378-1627-11e8-8eda-8f8a904326b4.png)
I think that if that object is intended to go in a JSON file it should be

```
{
  "viewportWidth": 1440,
  "viewportHeight": 900
}
```

Instead of:
```
{
  viewportWidth: 1440,
  viewportHeight: 900
}
```

Easy for devs but a QA was struggling with it 😅

**Comment by pgroot91** [comment]
Awesome 😂

**Comment by goncy** [comment]
Here we have @pgroot91 the QA guy who was struggling with it haha

**Comment by brian-mann** [comment]
Released in [`2.0.3`](https://docs.cypress.io/guides/references/changelog.html#2-0-3).
